packages/react/src/SelectPanel/SelectPanel.tsx

```diff
@@ -378,6 +382,37 @@ function Panel({
     }
   }, [open, dataLoadedOnce, onFilterChange, filterValue, items, loadingManagedExternally, listContainerElement])
 
+  useEffect(() => {
+    if (window.visualViewport) {
+      initialHeightRef.current = window.visualViewport.height
+    }
```

Comment:
I'm wondering if we should also check if the SelectPanel is open or not? This way, `handleViewportChange` will only run when it is open, and will clean up when closed. Could be something like this:

```suggestion
    if (!window.visualViewport || !open) {
      return
    }
    
    initialHeightRef.current = window.visualViewport.height
```

We'd need to add `open` as a dependency though. Curious what you think 🤔

Reply:
Great suggestion, thank you! I'll make this change 👍